### ISSUE TITLE
[a11y]: ai-chat should use descriptive label to convey purpose of chat input field

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#880]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

pon receiving focus -The screen reader does not recognize or announce the label of the element, making it unclear what the purpose of the control is.

Recommended fix: Screen reader expected to announce the Descriptive Label which defines the purpose of the control :
Ensure that all interactive elements, such as buttons, form fields, and links, have a programmatically associated and descriptive label using appropriate Technique, (HTML (e.g., label with for attribute , or aria-label, ,aria-labeled) so that screen readers can correctly identify those controls, for example: "Ask your question"

User impact: Impact all users group especially Screen reader users and People who use Voice control Tools to access forms :When labels are missing or not programmatically associated with form fields or controls, users cannot identify the purpose of the element. This leads to confusion, increased likelihood of input errors, difficulty completing tasks independently.

<img width="1920" height="965" alt="Image" src="https://github.com/user-attachments/assets/bce718bd-a91d-4514-8693-e741a1cd58af" />

### WCAG 2.1 Violation

WCAG Mapping: 3.3.2 Labels or Instruction

### Reproduction/example

n/a

### Steps to reproduce

1. make sure JAWS is active
2. open ai-chat
3. navigate to submit button
4. see that a meaningful description for the submit button is missing.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
